terraform/vpc/outputs.tf

```diff
@@ -0,0 +1,24 @@
+output "environment" {
+  value = "${var.env}"
+}
+
+output "region" {
+  value = "${var.region}"
+}
+
+output "vpc" {
+  value = "${aws_vpc.myvpc.id}"
+}
+
+output "vpc-CIDR" {
```

Comment:
I think it is better to keep this output name in lowercase, don't  you think so? This way we keep consistency.